### ISSUE TITLE
A couple issues with the bundle.yaml file

### ISSUE THREAD

**Issue by Jamie-SA** [semanticarts/gist#746]
I noticed a couple of problems with but bundle.yaml file that should be fixed. Check the PR for specifics.

**Comment by rjyounes** [comment]
@Jamie-SA We haven't been including `Contributing.md` and the style guide in the release package, though we've been including references to them in the README - however, the README was not getting into the release package. I've now added the following to my PR #712:

- Add `README.md` and `README.html` to the Documentation folder of the release package via changes to `bundle.yaml`.
- Update the `README` to include the contents of `gistDocumentation.md`. The former included parts but not all of the latter.
- Remove `gistDocumentation.md` from the repository, since it's now redundant.

These changes should have been moved to a new PR, but I started working on them and got too far in (also being short of time). 

Because we haven't in the past included `Contributing` or the style guide, I'm moving this issue to the next release for group discussion.
